src/pages/AboutUsPage/ServiceCard.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode } from "react";
+
+// Define an interface for the props
+interface ServiceCardProps {
+    icon: ReactNode;
+    title: string;
+    details: string;
+}
+
+const ServiceCard: React.FC<ServiceCardProps> = ({ icon, title, details }) => {
```

Comment:
we need to have one style of function change function to the same which we have in other components without React calling